W/Wayland/build_tarballs.jl

```diff
@@ -48,7 +54,6 @@ dependencies = [
     Dependency("Libffi_jll"; compat="~3.2.2"),
     Dependency("XML2_jll"),
```

Comment:
Is this package required for Wayland itself, or is this only a dependency for `wayland-scanner`?

Reply:
I think the XML packages are required only for wayland-scanner, but this is exposed as an ExecutableProduct